Resources/Textures/_Impstation/Clothing/Eyes/Hud/syndisec.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by IntegerTempest",
```

Comment:
you arent IntegerTempest, right? if you sourced this art asset from somewhere else, could you provide a link? asking in part because i cannot source this sprite anywhere